stl/inc/xutility

```diff
@@ -5563,17 +5563,36 @@ _NODISCARD constexpr auto lexicographical_compare_three_way(
 }
 #endif // __cpp_lib_concepts
 
+template <class _Ty, class _Elem>
+struct _Vector_alg_in_find_is_safe_object_pointers : false_type {};
+template <class _T1, class _T2>

```

Comment:
Although everything that's leading-underscore-capital is reserved, we conventionally avoid identifiers that look like macros - i.e. those that contain no lowercase letters. Therefore we should conventionally say `_Ty1` and `_Ty2` here.

(There are a few low-priority violations of this rule that are on my todo list to clean up someday.)